results/intfloat__multilingual-e5-small/IndonesianMongabayConservationClassification.json

```diff
@@ -0,0 +1,21 @@
+{
+  "dataset_revision": "c9e9f2c09836bfec57c543ab65983f3398e9657a",
+  "mteb_dataset_name": "IndonesianMongabayConservationClassification",
+  "mteb_version": "1.7.58",
+  "test": {
+    "accuracy": 0.2632548618219038,
+    "accuracy_stderr": 0.028227484443608907,
+    "evaluation_time": 13.28,
+    "f1": 0.26091359988519114,
```

Comment:
If there are only 3 classes, this seems to be below random chance. Wondering if you have more insights into that. (e.g. class imbalance).

Reply:
Hi @isaac-chung , yeah I think it's definitely class imbalance. When I checked there are more positive / neutral classes than that of negative ones.